custom/enikshay/private_sector_datamigration/management/commands/create_cases_by_beneficiary.py

```diff
@@ -167,3 +225,23 @@ def handle(self, domain, **options):
         logger.info('Setting cleanliness flags')
         set_cleanliness_flags_for_domain(domain, force_full=True, raise_soft_assertions=False)
         logger.info('Done!')
+
+    @staticmethod
+    def perform_checks(beneficiaries):
```

Comment:
Could these assertions be setup as `CHECK` constraints in the database? That way you'd know immediately when code does something that violates any of them.